troubleshooting/troubleshooting-cannot-receive-push-notifications-on-android-when-the-app-is-closed.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Cannot Receive Push Notifications on Android When the App Is Closed
+description: On Android, push notifications are not received when the app is not running (closed). Notifications are received when the app is running either in the foreground or in the background.  
+type: troubleshooting
+page_title:  Push Notifications are not received on Android When the App Is Closed 
```

Comment:
Title style capitalization, please. 
Push Notifications Are Not Received on Android When the App Is Closed